views/default/admin_domains_meta.pdt

```diff
@@ -0,0 +1,224 @@
+        <?php
+        echo (isset($message) ? $message : null);
+
+        $this->Widget->clear();
+        $this->Widget->setStyleSheet($this->view_dir . 'css/styles.css', ['id' => 'domains_styles']);
+        $this->Widget->create($this->_('AdminDomains.meta.boxtitle_meta_tld', true, $tld->tld), ['id' => 'meta_tld' . str_replace('.', '_', $tld->tld)], (isset($render_section) ? $render_section : null));
+
+        $this->Form->create(null, ['id' => 'meta_tld']);
+        ?>
+
+        <div class="inner">
+            <div class="modal_page" id="page_warning">
+                <div class="p-3">
+                    <h1><i class="fab fa-whmcs fa-fw"></i> <?php $this->_('AdminDomains.meta.heading_update_required');?></h1>
+                    <p class="text-black-50"><?php $this->_('AdminDomains.meta.text_update_required_note');?></p>
+                </div>
+
+                <div class="button_row">
+                    <a href="#" id="continue" class="btn btn-success float-right"><i class="fas fa-arrow-right"></i> <?php $this->_('AdminDomains.meta.field_continue');?></a>
+                </div>
+            </div>
+
+            <div class="modal_page" id="page_meta" style="display: none;">
+                <?php
+                if (!empty($package_fields['fields'])) {
```

Comment:
This page can be simplified using the new Fields/Html utility (Blesta\Core\Util\Input\Fields\Html).  For an example you can look at admin_packages.php and admin_packages_moduleoptions.pdt

Reply:
Updated